### ISSUE TITLE
libraries/math/helpers.cairo::to_uint256() function refactor

### ISSUE THREAD

**Issue by kaliberpoziomka** [aave-starknet-project/aave-starknet-core#97]
Currently function `to_uint256()` in `libraries/math/helpers.cairo` uses `assert_nn()` function to check argument `value`. This approach may not be wanted, since it allows us to convert `felt` to `Uint256` type only if `value` is in range: ` 0 <= value < RANGE_CHECK_BOUND`.

**Comment by EdgarBarrantes** [comment]
Agreed, do you think it's necessary to use something to check for negative values at all considering how `split_felt()` works?

**Comment by kaliberpoziomka** [comment]
I don't think we should check for negative values at all if we use `felt` as an unsigned integer
